### ISSUE TITLE
Command for getting initial rancher password doesn't work

### ISSUE THREAD

**Issue by thehejik** [rancher/dashboard#3575]
Seen on `rancher/rancher:master-head 94bca99`

Related to https://github.com/rancher/dashboard/issues/3574

The command for getting initial rancher password from `Welcome to Rancher` page doesn't work:
```bash
> kubectl get secret --namespace cattle-system bootstrap-secret -o go-template={{.data.bootstrapPassword|base64decode}}{{"\n"}}
If 'base64decode}}{{
}}' is not a typo you can use command-not-found to lookup the package that contains it, like this:
    cnf base64decode}}{{
}}
Error from server (NotFound): secrets "bootstrap-secret" not found
```
I identified two problems so far:
1) kubectl command triggered from bash complains that `base64decode}}{{ }}` command doesn't exist - reason is that pipe `|` is not escaped:
2) secret `bootstrap-secret` in `cattle-system` ns doesn't exist (rancher deployed over helm)

**Comment by vincent99** [comment]
The go-template should be single-quoted, it got lost somewhere.

That's the namespace & name in the helm notes and [what's in the chart](https://github.com/rancher/rancher/blob/master/chart/templates/secret.yaml)..  You'd need to be using the new version of the chart from master, not 'rancher-latest' or similar.

If the notes don't look like [this](https://github.com/rancher/rancher/blob/master/chart/templates/NOTES.txt) (`helm get notes -n cattle-system rancher`) you don't have the updated chart which generates a secret.

**Comment by thehejik** [comment]
How can I get the new version of the chart from master? I'm afraid we have to wait for 2.6 RC, or?

**Comment by sowmyav27** [comment]
Waiting for an RC from master

**Comment by thehejik** [comment]
This is fixed in rancher/rancher:2.6.0-rc1 deployed by helm. The secret exists and the offered kubectl command returns a valid rancher password. Also the command is escaped correctly now.

![password](https://user-images.githubusercontent.com/12828077/128695218-ae8cb77a-396c-4959-afac-790635881ea2.png)


Close.
